### DISCUSSION TITLE
Quarkus GraphQL schema: Unknowing argument "..." on field "..." error

### DISCUSSION THREAD

**Discussion by Bilel-NEJI** [quarkusio/quarkus#24457] [Q&A]
Hi @phillip-kruger,
My schema looks like this:
![Pic-11](https://user-images.githubusercontent.com/84474060/159358481-6f659f3a-ffb3-465c-844a-a5239baed939.jpg)

When I execute this query with this argument (miDeviceId: 1), I have this error:

![Pic-10](https://user-images.githubusercontent.com/84474060/159358785-66501444-8499-43d1-b21a-e376de9efcf8.jpg)

how can I remove this error so I can filter in the graphql-ui ?

[a comment by an automated account is not shown]

**Reply by phillip-kruger** [reply]
How does your code look like?

**Reply by Bilel-NEJI** [reply]
https://github.com/Bilel-NEJI/mcs-thirdparty
